app/uk/gov/hmrc/trackingconsentfrontend/assets/src/common/pageHandler.ts

```diff
@@ -1,10 +1,11 @@
 import enableGtm from '../interfaces/enableGtm'
 import { UserPreferences } from '../../types/UserPreferences'
 import isFeatureEnabled from "../interfaces/isFeatureEnabled";
-import {enableTrackingConsent} from "../constants/featureNames";
+import { enableTrackingConsent } from "../constants/featureNames";
+import getGtmContainerId from "./getGtmContainerId";
 
 const pageHandler = (document: HTMLDocument, userPreference: UserPreferences, pageRenderer, containerId) => {
-    enableGtm(containerId)
+    enableGtm(getGtmContainerId() || containerId)
```

Comment:
This logic means the existing links to tracking.js and tracking/transitional.js will continue to work for now without a data attribute. At some point we should throw an error for teams not being explicit about the container as there is no default.

Reply:
@nat.carey suggested adding a deprecation notice in the console